dlt/destinations/athena/configuration.py

```diff
@@ -12,6 +12,8 @@ class AthenaClientConfiguration(DestinationClientDwhWithStagingConfiguration):
     credentials: AwsCredentials = None
     athena_work_group: Optional[str] = None
     aws_data_catalog: Optional[str] = "awsdatacatalog"
+    supports_truncate_command: bool = False
+    force_iceberg: Optional[bool] = False
```

Comment:
is this used just for testing? I do not see other reason. you can just set all table formats to iceberg to force it

Reply:
how would I do this? i want to be able to set an env var and then have our standard tests all run against athena iceberg

Reply:
OK! then keep it. I think other methods would be too complicated (ie patching some functions)